include/extractor/turn_instructions.hpp

```diff
@@ -31,37 +37,87 @@ enum class TurnInstruction : unsigned char
     AccessRestrictionPenalty = 129
 };
 
+// shiftable turns to left and right
+const constexpr bool shiftable_left[] = {false, false, true, true, true, false, false, true, true};
+const constexpr bool shiftable_right[] = {false, false, true, true, false, false, true, true, true};
```

Comment:
Its asserted for a different size than the number of actual instructions, since we do not shift all instructions